### ISSUE TITLE
Custom Info Window with Button

### ISSUE THREAD

**Issue by adappter** [ng2-ui/map#135]
Hello,
I want to ask you if it's possible to listen for a button click inside an custom info-window:

` <info-window id="infoWindow">
        <ion-label>[[type]]</ion-label>
        <span>[[address]]</span>
        <span>[[description]]</span>
        <button ion-button color="danger" (click)="setToInactive()">Deactivate</button>
    </info-window>`

That how my info-window looks like but setToInactive won't be called.

Hope you guys can help me.

**Comment by adappter** [comment]
No one?

**Comment by davingreen** [comment]
Bump. Same question:
`<info-window id="iw">
     <button md-button (click)="callbackTest('test')">Test</button>
 </info-window>`

Doesn't seem to do anything.

**Comment by adappter** [comment]
The problem ist, that the info-window scope is not the angular scope. that's why the button doesnt call the function. I don't have a workaround or idea for that.

**Comment by aitboudad** [comment]
Currently not supported, next week I'll try to work on it.

**Comment by davingreen** [comment]
@aitboudad So can we use (click) events within an info window now? Or is that declared not possible?

**Comment by aitboudad** [comment]
yep see the example https://github.com/ng2-ui/map/blob/master/app/map-components/simple-info-window.component.ts

**Comment by allenhwkim** [comment]
@aitboudad I have granted access to @ngui/map. Now you can publish it, I think.
Can you verify it?

```
map$ (master) npm team ls ngui:map
[
  "aitboudad",
  "allenkim"
]
map$ (master) npm access ls-packages aitboudad
{
  "ng-formly": "read-write",
  "ng-loading-bar": "read-write",
  "@ngui/map": "read-write"
}
```

**Comment by aitboudad** [comment]
@allenhwkim Thanks I'd give a try tomorrow!

**Comment by aitboudad** [comment]
published under `0.18.0`
